core/src/View/MDSpan/Kokkos_MDSpan_Accessor.hpp

```diff
@@ -0,0 +1,146 @@
+
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#ifndef KOKKOS_IMPL_PUBLIC_INCLUDE
+static_assert(false,
+              "Including non-public Kokkos header files is not allowed.");
+#endif
+
+#ifndef KOKKOS_MDSPAN_ACCESSOR_HPP
+#define KOKKOS_MDSPAN_ACCESSOR_HPP
+
+#include <Kokkos_Macros.hpp>
+#include <Kokkos_Concepts.hpp>
+#include <impl/Kokkos_Utilities.hpp>
```

Comment:
What are you using from these?
I think all you need is an include for `Impl::runtime_check_memory_access_violation()`

Reply:
Macros: KOKKOS_FUNCTION
Concepts: is_memory_space
Utilities: runtime_check_memory_access_violation

or?

Reply:
```suggestion
#include <Kokkos_Core_fwd.hpp> // runtime_check_memory_access_violation
```
`runtime_check_memory_access_violation` (without `ViewTracker` and `ViewMapping` argument) is defined in `Kokkos_Core_fwd.hpp` (which also includes `Kokkos_Macros.hpp`).